core/domain/learner_progress_services.py

```diff
@@ -25,6 +25,7 @@
 from core.platform import models
 
 import utils
+import feconf
```

Comment:
feconf should come before utils (alphabetize).

Reply:
Done!
Should I change the order of line 21 and 22? (because of the same reason)

Reply:
Good catch! Yes, please do. You can also remove the newline on line 27 since all these imports are Oppia ones.

Reply:
Done!